packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -81,14 +87,20 @@ export class MonacoEditorLanguageClientWrapper {
         this.serviceConfig.debugLogging = this.logger.isEnabled() && (this.serviceConfig.debugLogging || this.logger.isDebugEnabled());
     }
 
-    async start(userConfig: UserConfig, htmlElement: HTMLElement | null) {
+    /**
+     * Performs a full user configuration and the languageclient wrapper (if used) init and then start the application.
+     */
+    async initAndStart(userConfig: UserConfig, htmlElement: HTMLElement | null) {
         await this.init(userConfig);
-        await this.startNoInit(htmlElement);
+        await this.noInitJustStart(htmlElement);
     }
 
-    async startNoInit(htmlElement: HTMLElement | null) {
+    /**
+     * Does not perform any user configuration or other application init and just starts the application.
+     */
+    async noInitJustStart(htmlElement: HTMLElement | null) {
```

Comment:
Would `start` itself here be a better choice?

Reply:
Probably yes, looking at it the next day. 😆